### PR TITLE
Cleanups

### PR DESCRIPTION
**Pull request by SlejmUr**
Part first where I clean up things.
Added my version of LGC for completly separate from all other solution. Hash and stuff. (NOT IN USE currently)
Removed newest Json.net and use already existing one from Tarkov.
removed AssemblyInfo and used basic propertygroup in csproj
finalized Version numbers. (you must edit the Version inside it!)
now using Github first 8 character (from last commit, good for auto push but not good for self test)
YOU MUST HAVE INSTALLED GIT ON YOUR PC TO BUILD! sorry but now its mandatory

### CODE REVIEW

**Comment by SlejmUr** [comment]
it fixed the readme for discord stuff

File: Source/EssentialPatches/LGC.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        }
+
+        static readonly Dictionary<string, string> sha1_kv = new()
+        {
+            { "EscapeFromTarkov_BE.exe", "8566a2400a9d2f5123f439b30744bb52e169d6dd" },
+            { "UnityCrashHandler64.exe", "80cee5a8f66e8f4e6d5d3e2f277d9ec846e89f91" },
+            { "Uninstall.exe", "7646c0fc24ae35cd688808da240f27ce58973286" },
+            { "EscapeFromTarkov_Data/app.info", "de9a095b95ec4105a06e9bc0f3bba150902818a8" },
+            { "UnityPlayer.dll", "b88cecc9aef25b04c0c0c1ad98bfcf0da1c8ed8c" },
+            { "NLog/NLog.config", "b4839b8057276da84a59ece081403a73b272674d" }
```
Comment (samurailink3) [inline Source/EssentialPatches/LGC.cs:85]: While this does make the LGC basically bullet-proof, its also **HIGHLY DANGEROUS**. If someone opens the BSG-Launcher and Live Tarkov updates, these hashes will likely change, breaking SIT until we can patch it.

Comment (SlejmUr) [reply Source/EssentialPatches/LGC.cs:85]: This is not checking the live game for these hashes, it checking already existing files next to the copied game.
And check line 59-62. If we are on live, we quit.

**Comment by SlejmUr** [comment]
Changes made. The request made by: Trippy

File: Source/UI/RemoveScavModeButtonPatch.cs [hunk 2]
```diff
@@ -1,7 +1,6 @@
 ﻿using EFT.UI;
 using SIT.Tarkov.Core;
 using System.Reflection;
-using UnityEngine;
```
Comment (mihaicm93) [inline Source/UI/RemoveScavModeButtonPatch.cs:4]: Could you put this back as we are gonna use it to disable the Scav Button, Thank you

**Comment by trippyone** [comment]
can you separate the cleaning and Legal check in two separate PRs please. i will merge the cleaning but the legal check I will leave it up as a PR while we decide if we are going to use it or not.

thanks

**Comment by SlejmUr** [comment]
Can you merge and delete the file? Would be much better for both of us.

**Comment by trippyone** [comment]
reverted, will discuss about some of the changes to the versioning with the team first.

**Pull request merged**